packages/EDA/src/utils.ts

```diff
@@ -49,3 +49,13 @@ export function checkGeneratorSVMinputs(samplesCount: number, featuresCount: num
   if ((violatorsPercentage < PERCENTAGE_MIN) || (violatorsPercentage > PERCENTAGE_MAX))
     throw new Error(INCORERRECT_PERCENTAGE_MES);
 }
+
+// Check columns (ANOVA)
+export function checkColumns(columns: DG.ColumnList): void {
+  if (Array.from(columns.numerical).length < columns.length)
+    throw new Error('non numerical columns are selected');
+
+  if (columns.toList().some((col) => col.stats.missingValueCount > 0))
+    grok.shell.warning('Columns with missing values are selected, the calculation results may not be accurate.\
```

Comment:
What is meant by "may not be accurate"? This is not right

Reply:
When passing columns with empty values to wasm functions, the empty values are converted to very small values close to zero (e.g., 2.78^-30), which is not NULL and can affect the calculations.

Reply:
Clearly they should not be converted to small values